app/src/main/java/com/kickstarter/ApplicationGraph.java

```diff
@@ -18,15 +13,10 @@
   Environment environment();
   void inject(ApplicationLifecycleUtil __);
```

Comment:
the things that remain in `ApplicationGraph` will probably be here to stay so long as we use Dagger (until the end of time) since they are base classes which cannot have an environment built in